File: API/src/DTO/CreateBookingDTO.ts
// src/DTO/CreateBookingDTO.ts
export interface CreateBookingDTO {
  vehicleId: string;
  customerFirstName: string;
  customerLastName: string;
  customerEmail: string;
  customerLicenseId: string;
  customerMobile: string;
  startDate: string;  // ISO 8601
  endDate:   string;  // ISO 8601
}

File: CarRental/DTOs/CreateBookingDto.cs
namespace CarRental.DTOs;

public class CreateBookingDto
{
    public string VehicleId { get; set; }
    public string CustomerFirstName { get; set; }
    public string CustomerLastName { get; set; }
    public string CustomerEmail { get; set; }
    public string CustomerLicenseId { get; set; }
    public string CustomerMobile { get; set; }
    public string StartDate { get; set; } 
    public string EndDate { get; set; }
}
